src/WebApi/Application/UseCases/QuestionList/GetQuestionList/IOutputPort.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Domain.Models;
+using System.Collections.Generic;
+
+namespace Application.UseCases.QuestionList.GetQuestionList;
+public interface IOutputPort
```

Comment:
Missing empty line between the namespace and the member declaration

```suggestion

public interface IOutputPort
```

because we care. EZ fix, I know Visual Studio sometimes make these things difficult to see with the references. Don't get me wrong, happens to me all the time. 😅